<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter database name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter table name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDBName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTableName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Column Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditColumnName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDataType">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Integer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Text</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonAddColumn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Column</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreateTable">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResponse">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
